docs/classes/Command.md

```diff
@@ -0,0 +1,264 @@
+# Class: Command
```

Comment:
Moved to separate folder `docs`

Updated links in the `README.md` to this file as entry point for documentation